To: vim_dev@googlegroups.com
Subject: Patch 7.3.100
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.100 
Problem:    When using :normal v:count isn't set. 
Solution:   Call normal_cmd() with toplevel set to TRUE. 
Files:      src/ex_docmd.c 
    

*** ../vim-7.3.099/src/ex_docmd.c	2010-12-02 16:01:23.000000000 +0100
--- src/ex_docmd.c	2011-01-06 17:23:43.000000000 +0100
***************
*** 9310,9316 ****
  								  && !got_int)
      {
  	update_topline_cursor();
! 	normal_cmd(&oa, FALSE);	/* execute a Normal mode cmd */
      }
  }
  #endif
--- 9310,9316 ----
  								  && !got_int)
      {
  	update_topline_cursor();
! 	normal_cmd(&oa, TRUE);	/* execute a Normal mode cmd */
      }
  }
  #endif
*** ../vim-7.3.099/src/version.c	2011-01-08 14:45:57.000000000 +0100
--- src/version.c	2011-01-17 19:49:07.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     100,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
109. You actually read -- and enjoy -- lists like this.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
